docs/reference/changelog.md

```diff
@@ -2,6 +2,39 @@
 
 🛠 = Version contains a breaking change. View the list of all the breaking changes [here](https://github.com/n8n-io/n8n/blob/master/packages/cli/BREAKING-CHANGES.md).
 
+## n8n@0.166.0
+
+For a comprehensive list of changes, view the [commits](https://github.com/n8n-io/n8n/compare/n8n@0.165.1...n8n@0.166.0) for this version.<br />
+**Release date:** 2022-03-08
+
+### New nodes ✨
+
+<br />
+<Changelog node="n8n-nodes-base.odoo" title="Odoo" />
+
+### Enhanced nodes 🚀
+
+<br />
+<Changelog node="n8n-nodes-base.function" title="Function:" text="Added support for items without a json key."/>
+
+### Core functionality ⚙️
+
+- Added new environment variable `N8N_HIRING_BANNER_ENABLED` to enable/disable the hiring banner.
+- Fixed a bug preventing keyboard shortcuts from working as expected.
```

Comment:
Should these be under bug fixes?

Reply:
Afaik the bug fix section was only ever for node-specific bug fixes using the `<Changelog>` tag

Reply:
Ah ok. We probably ought to retitle the bugfixes section to "Node bugfixes" then? As it looks weird to have a load of bugfixes, followed by a bugfixes section